ruma-events/src/enums.rs

```diff
@@ -94,45 +92,26 @@ event_content_enum! {
     ]
 }
 
-/// Any basic event, one that has no (well-known) fields outside of `content`.
-pub type AnyBasicEvent = BasicEvent<AnyBasicEventContent>;
-
-/// Any ephemeral room event.
-pub type AnyEphemeralRoomEvent = EphemeralRoomEvent<AnyEphemeralRoomEventContent>;
-
-/// Any message event.
-pub type AnyMessageEvent = MessageEvent<AnyMessageEventContent>;
-
 /// Any message event stub (message event without a `room_id`, as returned in `/sync` responses)
 pub type AnyMessageEventStub = MessageEventStub<AnyMessageEventContent>;
```

Comment:
Ah, I guess we do need the marker trait implementations... For now.

Reply:
Are we planning on removing the `Any*EventContent` enums? An event that has a `prev_content` is still possible right?

Reply:
The `Any*EventContent` enums are needed when sending events. I don't think they're needed as the `C` in `GenericEvent<C>` though.